tests/data/MerchFactory.ts

```diff
@@ -68,6 +98,14 @@ export class MerchFactory {
     return MerchandiseItemOptionModel.merge(fake, substitute);
   }
 
+  public static fakeOptionMetadata(): MerchItemOptionMetadata {
```

Comment:
Should we just always pass in `type` here? Seems much simpler that way.

Reply:
I feel like it's fine as is, if someone needs metadata with a specific type it's cause they need multiple options, so `createOptionMetadata` can be used there